lib/mockexpress.js

```diff
@@ -112,12 +119,30 @@ var MockExpress = function(route) {
 				},
         status: function(status) {
           sideEffects.status = status;
-          callback(null, sideEffects);
-        },
+					return this;
+				},
+				sendStatus: function(status) {
+          sideEffects.status = status;
+					callback(null, sideEffects);
+				},
         end: function(end) {
           sideEffects.end = end;
           callback(null, sideEffects);
-        }
+				},
+				cookie: function(cookie) {
+          if(sideEffects.cookie) {
+						sideEffects.cookie.push(cookie);
+					} else {
+						sideEffects.cookie = [cookie];
+					}
+				},
+				setHeader: function(header) {
```

Comment:
It looks like `res.setHeader` is not part of the Express 4.x API? The correct API should be `res.set` or `res.header` ([see the API Reference](http://expressjs.com/en/api.html#res.set)).

In addition, like `cookies`, it needs to take multiple arguments / different arguments, i.e.
* It should support both `field` (as a string) and `value`
* It should support an object like in the example in the documents.

This will also need a corresponding test.